src/tennisSimulator/Player.java

```diff
@@ -0,0 +1,62 @@
+package tennisSimulator;
+
+public class Player implements Comparable<Player> {
+
+    public static final String MEN_ORGANIZATION = "ATP";
+    public static final String WOMEN_ORGANIZATION = "WTP";
+    public static final String MEN = "men";
+
+    private final String name;
+    private final String gender;
+    private final String abbreviatedName;
+    private final int rating;
+    private final String organization;
+    private boolean isServing = false;
+
+    public Player(String name, String gender, int rating) {
+        this.name = name;
+        this.gender = gender;
+        this.rating = rating;
+        String [] splitName = name.split(" ");
+        abbreviatedName = splitName[1].substring(0, 3).toUpperCase();
+        if (gender.equals(MEN)) {
```

Comment:
What if someone enters MAN in capital? use equalsIgnoreCase method instead and that wouldn't be a problem.
What is more, I think ternary operator would make this part of code cleaner.

Reply:
fixed